decidim-proposals/app/views/decidim/proposals/proposals/_endorsement_button.html.erb

```diff
@@ -0,0 +1,25 @@
+<%
+# @params (mandatory): proposal, from_proposals_list
+# @params (optional) : user_group, btn_label
+# avoid "undefined local variable" error
```

Comment:
Remove commented code

Reply:
@mrcasals sorry, it is not commented code, it is documentation of the expected "arguments" for this partial.

Reply:
Ah, I see! 👌  Maybe add a line saying "allowed params" or something like that?